packages/world/ts/recipes.ts

```diff
@@ -281,24 +286,1594 @@ export const recipes: Recipe[] = [
+    inputs: [
+      ["Cotton", 8],
+      ["LimeDye", 1],
+    ],
+    outputs: [["LimeCotton", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Cotton", 8],
+      ["CyanDye", 1],
+    ],
+    outputs: [["CyanCotton", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Cotton", 8],
+      ["GrayDye", 1],
+    ],
+    outputs: [["GrayCotton", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Cotton", 8],
+      ["PurpleDye", 1],
+    ],
+    outputs: [["PurpleCotton", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Cotton", 8],
+      ["MagentaDye", 1],
+    ],
+    outputs: [["MagentaCotton", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Cotton", 8],
+      ["LightBlueDye", 1],
+    ],
+    outputs: [["LightBlueCotton", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Cotton", 8],
+      ["LightGrayDye", 1],
+    ],
+    outputs: [["LightGrayCotton", 8]],
+  },
+
+  // Colored terracotta recipes
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["BrownDye", 1],
+    ],
+    outputs: [["BrownTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["OrangeDye", 1],
+    ],
+    outputs: [["OrangeTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["WhiteDye", 1],
+    ],
+    outputs: [["WhiteTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["YellowDye", 1],
+    ],
+    outputs: [["YellowTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["RedDye", 1],
+    ],
+    outputs: [["RedTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["CyanDye", 1],
+    ],
+    outputs: [["CyanTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["BlackDye", 1],
+    ],
+    outputs: [["BlackTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["PurpleDye", 1],
+    ],
+    outputs: [["PurpleTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["BlueDye", 1],
+    ],
+    outputs: [["BlueTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["MagentaDye", 1],
+    ],
+    outputs: [["MagentaTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["LightGrayDye", 1],
+    ],
+    outputs: [["LightGrayTerracotta", 8]],
+  },
+  {
+    station: "Workbench",
+    inputs: [
+      ["Terracotta", 8],
+      ["LightBlueDye", 1],
+    ],
+    outputs: [["LightBlueTerracotta", 8]],
+  },
+  // Functional objects recipes
+  {
+    station: "Workbench",
+    inputs: [["Glass", 3]],
+    outputs: [["GlassPane", 8]],
```

Comment:
do we have a way to represent a `GlassPane` in the client @dhvanipa?

Reply:
yup should be